filestack/mixins/imagetransformation.py

```diff
@@ -126,6 +126,9 @@ def pdf_info(self, colorinfo=None):
     def pdf_convert(self, pageorientation=None, pageformat=None, pages=None):
         return self._add_transform_task('pdfconvert', locals())
 
+    def minify_css(self,level=None, gzip=None):
```

Comment:
Missed space between `self` and `level=None`

Reply:
this is done